src/wooting-usb.c

```diff
@@ -46,8 +44,50 @@ static uint16_t getCrc16ccitt(const uint8_t* buffer, uint16_t size)
 	return crc;
 }
 
-void wooting_usb_disconnect(bool trigger_cb) {
+#pragma region Set Meta
```

Comment:
What is this for?

Reply:
Is just a minor thing to group up those meta struct related functions into a single region. Only real thing it does is allows your editor to collapse them all